spec/controllers/api/accounts_controller_spec.rb

```diff
@@ -322,6 +323,75 @@
       expect(response).to have_http_status(200)
     end
 
+    it 'updates account and correctly move items' do
+      login_as(:bob)
+
+      account = accounts(:account1)
+      new_group = groups(:group2)
+
+      account_params = {
+        data: {
+          id: account.id,
+          attributes: {
+            accountname: 'Bob Meyer',
+            tag: 'taggy',
+            cleartext_username: 'globi',
+            cleartext_password: 'petzi'
+          },
+          relationships: { group: { data: { id: new_group.id, type: 'groups' } } }
+        }, id: account.id
+      }
+      patch :update, params: account_params, xhr: true
+
+      account.reload
+
+      plaintext_team2_password = teams(:team2).decrypt_team_password(bob, private_key)
+      account.decrypt(plaintext_team2_password)
+      item = account.items.first
+      item.decrypt(plaintext_team2_password)
+
+      expect(account.cleartext_username).to eq 'globi'
+      expect(account.cleartext_password).to eq 'petzi'
+      expect(item.cleartext_file).to eq 'Das ist ein test File'
+
+      expect(response).to have_http_status(200)
+    end
+
+    it 'updates account but does not move without team membership' do
+      login_as(:alice)
+
+      account = accounts(:account1)
+      new_group = groups(:group2)
+
+      account_params = {
+        data: {
+          id: account.id,
+          attributes: {
+            accountname: 'Bob Meyer',
+            tag: 'taggy',
+            cleartext_username: 'globi',
+            cleartext_password: 'petzi'
+          },
+          relationships: { group: { data: { id: new_group.id, type: 'groups' } } }
+        }, id: account.id
+      }
+      expect do
+        patch :update, params: account_params, xhr: true
+      end.to raise_error(RuntimeError, 'user is not member of new team')
```

Comment:
shouldn't we return a 403 here ?

Reply:
depends how we want to handle this. Someone who's not in the target team can still update the other attributes of the account. But can't move it.

Reply:
aaah, I see. this exception comes from the AccountMoveHändler. leave it this way!